18F/identity-idp
# LG-12713: Remove reCAPTCHA checkbox fallback

## 🎫 Ticket

[LG-12713](https://cm-jira.usa.gov/browse/LG-12713)

## 🛠 Summary of changes

Removes the reCAPTCHA checkbox screen as a fallback when failing the invisible analysis, and replaces it with an error message informing the user that we cannot process their request.

## 📜 Testing Plan

Verify that you see an error message after submitting phone setup with a failing reCAPTCHA score:

Prerequisite: Configure score threshold in `config/application.yml`:

```
phone_recaptcha_score_threshold: 0.6
```

1. Go to http://localhost:3000
2. Sign in or create an account
3. Add a phone as an MFA method (select phone during MFA selection in account creation, or choose "Add phone number" from account dashboard after signing in)
4. Enter an international phone number (e.g. `+610491570006`)
5. In "reCAPTCHA score: (0.0 - 1.0) Internal Only" debugger, enter a score below the configured threshold (`0.4`)
6. Click "Send code"
7. Observe error message (refer to screenshots below)

## 👀 Screenshots

Before|After
---|---
![image](https://github.com/18F/identity-idp/assets/1779930/cd8d81c3-74fd-411c-b684-35e51df2ffd3)|![image](https://github.com/18F/identity-idp/assets/1779930/e55a8eb5-5046-4c38-a00d-530787818c2d)
